docs/APK_USER_DOCUMENTATION.md

```diff
@@ -0,0 +1,40 @@
+<!--
+
+    Sonatype Nexus (TM) Open Source Version
+    Copyright (c) 2019-present Sonatype, Inc.
+    All rights reserved. Includes the third-party code listed at http://links.sonatype.com/products/nexus/oss/attributions.
+
+    This program and the accompanying materials are made available under the terms of the Eclipse Public License Version 1.0,
+    which accompanies this distribution and is available at http://www.eclipse.org/legal/epl-v10.html.
+
+    Sonatype Nexus (TM) Professional Version is available from Sonatype, Inc. "Sonatype" and "Sonatype Nexus" are trademarks
+    of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
+    Eclipse Foundation. All other trademarks are the property of their respective owners.
+
+-->
+## APK Repositories
+
+### Introduction
+
+[APK](https://wiki.alpinelinux.org/wiki/Alpine_Linux_package_management) is the built-in package manager for Alpine Linux, a security-oriented, lightweight Linux distribution based on musl libc and Busybox.
+
+### Proxying APK repository
+
+You can create a proxy repository in Nexus Repository Manager (NXRM) that will cache packages from a remote APK repository such as
+http://dl-cdn.alpinelinux.org/alpine/v3.10/main/. To make `Alpine` use your NXRM Proxy, you will need to edit the file located at `/etc/apk/repositories`. More information can be found [here](https://wiki.alpinelinux.org/wiki/Enable_Community_Repository). 
+ 
+To proxy a APK repository, you simply create a new 'apk (proxy)' as documented in 
+[Repository Management](https://help.sonatype.com/repomanager3/configuration/repository-management). 
+
+Minimal configuration steps are:
```

Comment:
Trivial: I suspect this is just copyover maybe exists everywhere, but it's weird to me Name has a - and URL doesn't before e.g.
Also FYI, blobstore is required as well, but it auto defaults to "default" if you only have 1 so you never see it.  This is something I've added to every single community h.s.c page we've created (you can see for example on https://help.sonatype.com/display/NXRM3/Bower+Repositories#BowerRepositories-ProxyingBowerRepositories) in case you want to stop the madness ;)  Also trivial tho, most folks will find this out through validation if needed.

Reply:
@joedragons just checking was there suppose to be a screen shot or snippet after `e.g.`? I am trying to see the `-` usage you are talking about but can't seem to find it

Reply:
@joedragons woof, need to get my eyes check. I see what you mean now.

in: https://github.com/sonatype-nexus-community/nexus-repository-apk/pull/1/commits/244569afd7103fed515037840f3307541975fb85

Reply:
Heh, sorry, awkward line break, my bad=)